### DISCUSSION TITLE
How to make video full screen in WebView?

### DISCUSSION THREAD

**Discussion by NazimMertBilgi** [NativeScript/NativeScript#9641] [Q&A]
How to make video full screen in Android WebView? 
this could be any video. It doesn't react when I press the full screen key.

**Comment by NazimMertBilgi** [comment]
@NathanaelA @bradmartin @triniwiz

**Comment by febriarief** [comment]
any update ?

**Comment by farfromrefug** [comment]
This is how it is done in android https://stackoverflow.com/questions/15768837/playing-html5-video-on-fullscreen-in-android-webview/56186877#56186877
You won't integrate that easily in N webview. You can use @nativescript-community/ui-webview which allows you to set a custom client.
